src/patternfly/components/Drawer/drawer.scss

```diff
@@ -33,58 +43,174 @@ $pf-c-drawer__panel--width-list: (25, 33, 50, 66, 75);
+    --pf-c-drawer__content-body--m-padding--PaddingBottom: var(--pf-c-drawer__content-body--m-padding--md--PaddingBottom);
+    --pf-c-drawer__content-body--m-padding--PaddingLeft: var(--pf-c-drawer__content-body--m-padding--md--PaddingLeft);
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--xl) {
+    --pf-c-drawer__panel--FlexBasis: var(--pf-c-drawer__panel--xl--FlexBasis);
+    --pf-c-drawer__panel--MinWidth: var(--pf-c-drawer__panel--xl--MinWidth);
   }
 
   // Box shadow
   --pf-c-drawer__panel--BoxShadow: none;
-  --pf-c-drawer--m-expanded__panel--BoxShadow: var(--pf-global--BoxShadow--md-left);
-  --pf-c-drawer--m-expanded--m-panel-left__panel--BoxShadow: var(--pf-global--BoxShadow--md-right);
+  --pf-c-drawer--m-expanded__panel--BoxShadow: var(--pf-global--BoxShadow--lg-left);
+  --pf-c-drawer--m-expanded--m-panel-left__panel--BoxShadow: var(--pf-global--BoxShadow--lg-right);
 
   // Divider
-  --pf-c-drawer--m-divider__panel--after--Width: var(--pf-global--BorderWidth--sm);
-  --pf-c-drawer--m-divider--m-expanded__panel--after--BackgroundColor: var(--pf-global--BorderColor--100);
+  --pf-c-drawer__panel--after--Width: var(--pf-global--BorderWidth--sm);
+  --pf-c-drawer__panel--after--BackgroundColor: transparent;
+  --pf-c-drawer--m-inline--m-expanded__panel--after--BackgroundColor: var(--pf-global--BorderColor--100);
+
+  // Simple list
+  --pf-c-drawer--c-simple-list__title--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-drawer--c-simple-list__title--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-drawer--c-simple-list__title--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-drawer--c-simple-list__title--PaddingLeft: var(--pf-global--spacer--lg);
+
+  // Simple list item
+  --pf-c-drawer--c-simple-list__item-link--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-drawer--c-simple-list__item-link--PaddingLeft: var(--pf-global--spacer--lg);
+
+  // Action
+  --pf-c-drawer__action--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-drawer__panel-body--c-drawer__action--MarginRight: calc(var(--pf-global--spacer--md) * -1);
 
   display: flex;
+  flex-direction: column;
+  min-width: var(--pf-c-drawer__panel--MinWidth);
   height: 100%;
   overflow-x: hidden;
+
+  .pf-c-simple-list__title {
+    --pf-c-simple-list__title--PaddingTop: var(--pf-c-drawer--c-simple-list__title--PaddingTop);
+    --pf-c-simple-list__title--PaddingRight: var(--pf-c-drawer--c-simple-list__title--PaddingRight);
+    --pf-c-simple-list__title--PaddingBottom: var(--pf-c-drawer--c-simple-list__title--PaddingBottom);
+    --pf-c-simple-list__title--PaddingLeft: var(--pf-c-drawer--c-simple-list__title--PaddingLeft);
+  }
+
+  .pf-c-simple-list__item-link {
+    --pf-c-simple-list__item-link--PaddingRight: var(--pf-c-drawer--c-simple-list__item-link--PaddingRight);
+    --pf-c-simple-list__item-link--PaddingLeft: var(--pf-c-drawer--c-simple-list__item-link--PaddingLeft);
```

Comment:
I think with the title and link padding, we can make the same change as I mentioned below re: the toolbar's default left/right padding. Have the spacing adjust with the page gutter system so it's 24px on desktop and 16px on mobile. And that could go in the simple list, so we shouldn't need to update it here.